deploy/bundle/manifests/ocs-operator.clusterserviceversion.yaml

```diff
@@ -2420,6 +2420,8 @@ spec:
                   value: "true"
                 - name: CSI_VOLUME_REPLICATION_IMAGE
                   value: quay.io/csiaddons/volumereplication-operator:v0.1.0
+                - name: ROOK_CSIADDONS_IMAGE
```

Comment:
imo, its better to keep this variable as `ROOK_CSIADDONS_K8S_IMAGE` , considering CSI ADDONS got various images in place.

Reply:
this is the image name we are using in rook. we don't use k8s not sure where it's coming from